src/app-layout/visual-refresh-toolbar/drawer/local-drawer.tsx

```diff
@@ -71,6 +71,7 @@ export function AppLayoutDrawerImplementation({ appLayoutInternals }: AppLayoutD
           aria-hidden={!activeDrawer}
           aria-label={computedAriaLabels.content}
           className={clsx(styles.drawer, sharedStyles['with-motion'], {
+            [styles['drawer-hidden']]: !activeDrawer,
```

Comment:
This css class is not needed here as this drawer in hidden state has `0px` width